terraform/az-management/main.tf

```diff
@@ -2,23 +2,25 @@ provider "aws" {
   region = var.region
 }
 
-data "aws_subnet_ids" "selected" {
-  vpc_id = var.vpc_id
-
+data "aws_subnets" "selected" {
```

Comment:
It's a very minor thing, but ease of comprehension could we call this `all_subnets` and rename `subnet_ids` to `selected_subnet_ids`. Having this named `selected` and not choose all of the ones from it is a mite misleading.

Reply:
Could it be confusing to say "all"? I thought they were the ones for a particular availability zone

I'm not a fan or repeating the resource type in the name (see [terraform naming conventions](https://www.terraform-best-practices.com/naming)) - but I guess we're not sticking to the conventions anyways so that's not a bit deal 😅

Reply:
`all_subets_in_az` would be OK too

Reply:
https://en.wiktionary.org/wiki/subet I'll let you dig into that yourself

Reply:
I'm a bit confused over what I'm supposed to do - there _isn't_ a `subnet_ids` identifier anymore, other than the `aws_network_acl` attribute, which we don't get to choose the name of.

Reply:
Oh bloody 'ell, that'll teach me to not expand the diff. I read that attribute as being a variable. Sorry folks, false alarm! Leave it as is if we can't change it.